packages/render-ui/src/preview-prompt.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
```

Comment:
add a snapshot test? `expect( ...).toMatchSnapshot()` - you'll see examples in the repo